### ISSUE TITLE
Adding "Go to declaration" feature

### ISSUE THREAD

**Issue by jacopoabramo** [puorc/awesome-vhdl#9]
Hi,

I'm using your extension a lot for my VHDL and I find it quite pleasing, so first of all I wanted to thank you for your work. I wanted to ask if it was possible to add a "Go to/Peek declaration" feature. I find it quite useful since I sometimes get lost within the file and having a quick way to find a signal/component/whatever instead of doing CTRL + F is particularly useful.

**Comment by GlenNicholls** [comment]
@jethro33 I do not think this extension has the ability to do this. However, I've been using rust_hdl and it does have this ability!
